src/devices/src/virtio/block/event_handler.rs

```diff
@@ -49,3 +49,26 @@ impl Subscriber for Block {
         ]
     }
 }
+
+#[cfg(test)]
+mod tests {
+
+    use super::super::device::tests::default_block;
+    use super::*;
+
+    #[test]
+    fn test_interest_list() {
+        let block = default_block();
+        let interest_list = block.interest_list();
+        assert_eq!(interest_list.len(), 2);
+
+        let fds = vec![block.rate_limiter.as_raw_fd(), block.queue_evt.as_raw_fd()];
+        for i in 0..2 {
+            assert_eq!(interest_list[i].fd(), fds[i]);
+            assert_eq!(
+                EventSet::from_bits(interest_list[i].events()).unwrap(),
+                EventSet::IN
+            );
+        }
```

Comment:
This test should be removed as it doesn't provide any useful checks. `interest_list()` just returns a static vector.

Reply:
Done.